### ISSUE TITLE
recommend landscape hint

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#663]
requested by @abrasax for the treebank and usage examples panels, perhaps we could add a hint to the user to switch to landscape mode.  

Ideally we would switch them automatically, but based upon earlier tests facilitated by @kirlat , this wasn't universally supported across all devices.

**Comment by balmas** [comment]
@irina060981 here's a request for a hint :-)

I'm not sure how this should look. Maybe format the text using the important notification style, and put it at the top of the tab when in portrait mode on mobile? Probably should add it to inflections too.

**Comment by irina060981** [comment]
@balmas, I wonder, what exactly we want to tell to the user on the hint?
If a user simply rotates a device - it will look nlike this:

![image](https://user-images.githubusercontent.com/12377640/62191064-1dfc3b00-b3b6-11e9-8fb5-8b67b6f61e8f.png)

A user should also expand the panel?
What do you think?

**Comment by balmas** [comment]
yes, you're right.  How about this text, as a closeable notification?

"🛈 This view is best in landscape mode with the panel expanded to full screen"

**Comment by irina060981** [comment]
>"🛈 This view is best in landscape mode with the panel expanded to full screen"

I like it :) Will continue to work here

**Comment by irina060981** [comment]
![image](https://user-images.githubusercontent.com/12377640/62532200-3a511980-b887-11e9-938c-afed76c7faed.png)

I have added a check - to show hint only once for each tab - `treebank`, `inflections`, `inflectionsbrowser`

**Comment by irina060981** [comment]
Merged and closed

**Comment by balmas** [comment]
components 1.2.20

**Comment by monzug** [comment]
we are missing the hint from usage example

**Comment by monzug** [comment]
also, I am not able to test this issue on Samsung 5 mini as the morphology does not load any icons, except for close and move up. any ne else having trouble on Samsung?

**Comment by balmas** [comment]
hmmm that problem with the Samsung mini should have been fixed with #580. Any screen under 360px they should be hidden for, but I the Samsung mini is 360px isn't it?

**Comment by monzug** [comment]
I am not in landscape view, I checked the #580 and I tested on the Samsung and was ok.

**Comment by balmas** [comment]
>I am not in landscape view, I checked the #580 and I tested on the Samsung and was ok.

Not sure I am understanding. It was working before, but not now?

**Comment by balmas** [comment]
this is what I get in the debugger with the size set to what the Galaxy 5 should have (360x640)
![AwesomeScreenshot-texts-test-alpheios-net-text-urn_cts_latinLit_phi0474 phi004 perseus-lat2-passage-1-5-2019-08-07_10_23](https://user-images.githubusercontent.com/644870/62630736-83dc4b80-b8fd-11e9-94c6-6fb355bb557f.png)

**Comment by monzug** [comment]
I want to check if the hint is generated and is not in my Samsung but could be connection issue.
anyhow, the problem with this issue is what I reported above: hint not generated for usage example.
when I close the hint, is it  for the session right?

**Comment by balmas** [comment]
yes I think it only shows it once per tab per session

**Comment by monzug** [comment]
I install the new version of Chrome and morphology is generated on Samsung mini and I could also see the hints. confirm that no hint is generated for usage examples also on samsung

**Comment by balmas** [comment]
components 1.2.21

**Comment by monzug** [comment]
yes, hint is there both on Samsung and iPhone xr. thanks
